cli/command/load_balancer_test.go

```diff
@@ -0,0 +1,40 @@
+package command
+
+import (
+	"testing"
+
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/urfave/cli"
+)
+
+func TestCreateLoadBalancer(t *testing.T) {
```

Comment:
We should add another `TestCreateLoadBalancerDefaults` that ensure all the default arguments get passed correctly.